gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * (c) Copyright 2017 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.plugins;
+
+import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
+import java.io.File;
+import java.util.List;
+import java.util.stream.Collectors;
+import net.ltgt.gradle.errorprone.ErrorPronePlugin;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.artifacts.Configuration;
+import org.gradle.api.plugins.JavaPluginConvention;
+import org.gradle.api.tasks.compile.JavaCompile;
+import org.gradle.api.tasks.javadoc.Javadoc;
+import org.gradle.api.tasks.testing.Test;
+
+public final class BaselineErrorProne implements Plugin<Project> {
+
+    @Override
+    public void apply(Project project) {
+        project.getPluginManager().withPlugin("java", plugin -> {
+            JavaPluginConvention javaConvention = project.getConvention().getPlugin(JavaPluginConvention.class);
+
+            project.getPluginManager().apply(ErrorPronePlugin.class);
+            Configuration rawErrorProneConf = project.getConfigurations().getByName("errorprone").copy();
+            project.getConfigurations().getByName("errorprone")
+                    .defaultDependencies(dependencies -> dependencies.add(
+                            project.getDependencies().create(
+                                    "com.palantir.baseline:baseline-error-prone:latest.release")));
```

Comment:
This is a tiny bit subtle, as people could end up applying `com.palantir.baseline-error-prone` version 0.47.0 and getting rules from a newer release like 0.49.0...

Reply:
if you care about a version you should specify a version